test/suite/test_compact10.py

```diff
@@ -156,25 +156,27 @@ def test_compact10_full_backup(self):
     # comparison and another one as a base for incremental backups.
     # - Performs incremental backups as compaction is rewriting the files until all files have been
     # compacted.
-    # - Compares the two backups.
+    # - Compares all performed incremental backups with the original full backup.
     def test_compact10_incr_backup(self):
-
         # FIXME-WT-11399
         if self.runningHook('tiered'):
             self.skipTest("this test does not yet work with tiered storage")
 
+        backup_full = "BACKUP_FULL"
+        backup_incr = "BACKUP_INCR"
         uris = self.generate_data()
 
         # Take two full backups:
-
         # - The first one will remain untouched.
-        os.mkdir(self.backup_dir_1)
-        self.take_full_backup(self.backup_dir_1)
+        os.makedirs(backup_full)
+        self.take_full_backup(backup_full)
 
-        # - The second one is used for the incremental backups.
-        os.mkdir(self.backup_dir_2)
+        # - The second one is used for the incremental backups and use a temporary backup
+        # directory to perform multiple incremental backups on different directories.
         self.initial_backup = True
-        self.take_full_backup(self.backup_dir_2)
+        os.makedirs(backup_incr)
+        self.take_full_backup(backup_incr)
+        shutil.copytree(backup_incr, self.home_tmp)
```

Comment:
Can you add a comment why we are using `copytree` here?

Reply:
I have in the comment above it 
```
        # - The second one is used for the incremental backups and use a temporary backup
        # directory to perform multiple incremental backups on different directories.
```